storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.28-20.0.rst

```diff
@@ -0,0 +1,65 @@
+
+Release Highlights
+=================================================
+
+The following list contains a number of the notable updates and fixes for *MySQL* 8.0.28, provided by Oracle, and included in Percona XtraBackup for MySQL:
+
+* The GnuPG build key has been updated. The GnuPG build key is used to sign MySQL downloadable packages. The systems configured to use repo.mysql.com may report a signature verification error when upgrading using *apt* or *yum* to MySQL 5.7.37 and higher or to MySQL 8.0.28 and higher. To resolve this issue, use one of the following methods:
+
+    - Reinstall the MySQL APT or YUM repository setup package from https://dev.mysql.com/downloads/.
+
+    - Download the MySQL GnuPG public key and add it to your system GPG keyring.
+
+    - Find MySQL APT repository instructions in `Appendix A: Adding and Configuring the MySQL APT Repository Manually <https://dev.mysql.com/doc/mysql-apt-repo-quick-guide/en/#repo-qg-apt-repo-manual-setup>`__. 
+
+    - Find MySQL YUM repository instructions in `Upgrading MySQL with the MySQL Yum Repository <https://dev.mysql.com/doc/mysql-yum-repo-quick-guide/en/#repo-qg-yum-upgrading>`__. 
+* The following character sets and their collations are now deprecated and will be removed in a subsequent release of MySQL:
+
+    - `ucs2`
+
+    - `macroman` and `macce`
+
+    - `dec`
+
+    - `hp8`
+
+Use `utf8mb4` character set instead of any of the aforementioned character sets. 
+
+Find more information in `The ucs2 Character Set (UCS-2 Unicode Encoding) <https://dev.mysql.com/doc/refman/8.0/en/charset-unicode-ucs2.html>`__, `West European Character Sets <https://dev.mysql.com/doc/refman/8.0/en/charset-we-sets.html>`__, and `Central European Character Sets <https://dev.mysql.com/doc/refman/8.0/en/charset-ce-sets.html>`__.
+
+* As of this release, InnoDB supports `ALTER TABLE ... RENAME COLUMN` operations using `ALGORITHM=INSTANT`.
+
+Operations that support ALGORITHM=INSTANT only modify metadata in the data dictionary. During the preparation and execution of the operation, the exclusive metadata is not locked in the table. Also, the table data does not change which makes the operations instantaneous. DDL operations use `ALGORITHM=INSTANT` by default.
+
+Find more information about DDL operations that support `ALGORITHM=INSTANT` in `Online DDL Operations <https://dev.mysql.com/doc/refman/8.0/en/innodb-online-ddl-operations.html>`__.
+
+Find the full list of bug fixes and changes in the `MySQL 8.0.28 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-28.html>`__.
+
+Bugs Fixed
+==================================================
+
+* :jirabug:`PXB-2662`: Percona XtraBackup exited during prepare when creating an incremental backup using page tracking. 
+* :jirabug:`PXB-1676`: The size of the redo log was incorrect after creating a backup. 
+* :jirabug:`PXB-2701`: An incremental backup on MyRocks did not determine if the previous full or incremental backup had the same files.
```

Comment:
This is a documentation bug. We should remove it from release notes as they are not tied to a specific release.

Reply:
Removed